linux_docker_resources/Dockerfile

```diff
@@ -175,12 +175,6 @@ RUN useradd -u 1234 -m rosbuild
 RUN sudo -H -u rosbuild -- git config --global user.email "jenkins@ci.ros2.org"
 RUN sudo -H -u rosbuild -- git config --global user.name "Jenkins ROS 2"
 RUN echo 'rosbuild ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
-WORKDIR "@workdir"
-
-# Add an entry point which changes rosbuild's UID from 1234 to the UID of the invoking user.
-# This means that the generated files will have the same ownership as the host OS user.
-ADD entry_point.sh /entry_point.sh
-RUN chmod 755 /entry_point.sh
```

Comment:
I wouldn't be opposed to writing out a placeholder bash script that no-ops, or even just symlinking something in the script's spot. It would be nice to be able to `docker run` this as a human being without all of the extra bind mounts for debugging purposes.